src/IssueViz.Security/Taint/TaintList/ViewModels/TaintIssuesControlViewModel.cs

```diff
@@ -148,7 +147,24 @@ public ITaintIssueViewModel SelectedIssue
             ISonarQubeService sonarQubeService,
             INavigateToRuleDescriptionCommand navigateToRuleDescriptionCommand
             )
+            : this(store, locationNavigator, activeDocumentTracker, activeDocumentLocator, showInBrowserService, telemetryManager, selectionService,
+                  navigateToDocumentationCommand, menuCommandService, sonarQubeService, navigateToRuleDescriptionCommand, ThreadHandling.Instance)
+        { }
+
+        internal /* for testing */ TaintIssuesControlViewModel(ITaintStore store,
+            ILocationNavigator locationNavigator,
+            IActiveDocumentTracker activeDocumentTracker,
+            IActiveDocumentLocator activeDocumentLocator,
+            IShowInBrowserService showInBrowserService,
+            ITelemetryManager telemetryManager,
+            IIssueSelectionService selectionService,
+            ICommand navigateToDocumentationCommand,
+            IMenuCommandService menuCommandService,
+            ISonarQubeService sonarQubeService,
+            INavigateToRuleDescriptionCommand navigateToRuleDescriptionCommand,
+            IThreadHandling threadHandling)
```

Comment:
One of the code smells raised by SonarCloud for this PR is because this method has too many parameters (changed from 11 to 12, when the limit is 7). Fixing that would take quite a bit more refactoring.